var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -72,10 +84,19 @@ def install(self, spec, prefix):
                     '-DLIBELF_INCLUDE_DIR=%s'   % join_path(
                         libelf.include, 'libelf'),
                     '-DLIBELF_LIBRARIES=%s'     % join_path(
-                        libelf.lib, 'libelf.so'),
-                    '-DLIBDWARF_INCLUDE_DIR=%s' % libdwarf.include,
-                    '-DLIBDWARF_LIBRARIES=%s'   % join_path(
-                        libdwarf.lib, 'libdwarf.so')]
+                        libelf.lib, "libelf." + dso_suffix)]
+            if spec.satisfies('@:9.3.2'):
+                args.append('-DLIBDWARF_INCLUDE_DIR=%s' % libdwarf.include)
+                args.append('-DLIBDWARF_LIBRARIES=%s'   % join_path(
+                    libdwarf.lib, "libdwarf." + dso_suffix))
+            # For @develop + use elfutils libdw, libelf is an abstraction
+            # we are really using elfutils here
+            if spec.satisfies('@develop'):
+                args.append('-DLIBDWARF_INCLUDE_DIR=%s' % libelf.include)
+                args.append('-DLIBDWARF_LIBRARIES=%s'   % join_path(
+                    libelf.lib, "libdw." + dso_suffix))
+            if spec.satisfies('arch=linux-redhat7-ppc64le'):
+                args.append('-Darch_ppc64_little_endian=1')
```

Comment:
This line and the one above already exist in the 2 lines below.

Reply:
@lee218llnl I just submitted a change to remove those lines.   
My old branch didn't have the duplicate lines.   Not sure how that happened.